molecule/centos/molecule.yml

```diff
@@ -1,6 +1,7 @@
 ---
 dependency:
   name: galaxy
+  enabled: False
```

Comment:
Figured I disable the check since there are no deps and it produces a warning about a "missing requirements.yml".

Reply:
Makes sense